README.md

```diff
@@ -20,6 +20,17 @@ var db = new PouchDB('dbname')
 var api = db.hoodieApi()
 ```
 
+#### Options
+
+- emitter: Optionally pass an instance of EventEmitter for hoodieApi to use
+
+```js
+var db = new PouchDB('dbname')
+var api = db.hoodieApi({
+  emitter: existingEventEmitter
+})
+```
+
```

Comment:
Can we move this into the `### Api` block?

I'd put it in the same code block as the rest, on top of it?